mozilla_django_oidc_db/constants.py

```diff
@@ -11,3 +11,16 @@
 OPEN_ID_CONFIG_PATH = ".well-known/openid-configuration"
 
 CONFIG_CLASS_SESSION_KEY = "_OIDCDB_CONFIG_CLASS"
+
+CLAIM_MAPPING_SCHEMA = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Claim Mapping",
+    "description": "Mapping from user-model fields to OIDC claims",
+    "type": "object",
+    "properties": {},
+    "additionalProperties": {
+        "description": "mapping",
+        "type": "array",
```

Comment:
Is this correct? This reads like it expects somethings like this: 

```json
{
  "mapping": ["foo", "bar"]
}
```

but shouldn't it just be a mapping from strings to strings?

Reply:
Apparently it should be a list of strings instead of string to string: https://github.com/maykinmedia/mozilla-django-oidc-db/pull/115#discussion_r1765746173